manifests/charts/controlplane/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: controlplane
+description: Helm umbrella chart for Istio controlplane components
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+kubeVersion: ">= 1.23.0-0"
+keywords:
+  - istiod
+  - istio
+  - controlplane
+  - mesh
+sources:
+  - https://github.com/istio/istio
+icon: https://istio.io/latest/favicons/android-192x192.png
+
+# These will be stamped out to real versions during release publish
+dependencies:
+  # TODO it makes the most logical sense for `base`
+  # (which includes CP CRDs, mostly) to be installed
+  # with the controlplane. But, as it is a singleton chart,
+  # it cannot be, without some work.
+  # - name: base
+  #   version: 1.0.0
```

Comment:
I'm not sure what  this does - it loads istiod chart ( which we have documented and used for some time). Why do we need an alias and associated confusion ?

If there is anything missing in istiod chart - we should fix it, I agree with the user flow and separation of control/data plane - but not with arbitrary new names and having 2 ways to do effectively the same.

The fundamental issue here is the base - CRDs in particular - but Istiod should work without CRD installed and we can avoid any other dependency.

Reply:
> I'm not sure what this does - it loads istiod chart ( which we have documented and used for some time). Why do we need an alias and associated confusion ?

Yes, I mentioned this [here](https://github.com/istio/istio/pull/54265#discussion_r1907979488) - I also agree that we do not need an `istiod` wrapper chart if it only wraps the single chart, and would be fine dropping it so that the UX looks like

`helm install istio-control/istio-discovery`
`helm install istio-dataplane`


i.e. still 2 charts. 


I like the idea of having consistent naming (`istio-control/istio-discovery` is neither intuitive or discoverable) but that's a different problem.